src/main.cpp

```diff
@@ -239,11 +242,17 @@ int main(int argc, char *argv[])
     conf = new ConfFile();
     parse_conf_files(*conf, &opt);
 
+#ifdef ENABLE_MAVLINK
+    CameraServer camServer(*conf);
```

Comment:
As explained above, I plan to do other way round. Add streaming feature to CameraComponent.